### PR TITLE
Add dev docker-compose config, with lpdc service

### PR DESCRIPTION
**Pull request by piemonkey**
## Overview
Add an lpdc proxy running ~~on a custom port~~ a path, using nginx-proxy settings (we don't seem to have wildcard DNS domains, so can't use a subdomain without setting up a new record).

edit: The change to using a subroute means that we can now actually have https on it, as well as not publishing weird ports.

~~Draft until the api-proxy change is released.~~

##### connected issues and PRs:
Needs an 'api-proxy' image with support for setting allowed origin header: https://github.com/lblod/api-proxy-service/pull/3

### Setup
Locally should just work, as the change has been deployed and it will use this api. To test fully locally you'll need to add a compose.override with a valid key for lpdc, publish a port for lpdc-service and to modify the config of the lpdc plugin to point to this localhost port. ~~Since the api-proxy change hasn't been merged, you'll need to add an override pointing it to a local copy of the repo with the change:~~

### How to test/reproduce
Open the test app and go to either of the plugins pages or the multiple growing editors page and verify that `insert lpdc` loads the test data.

### Challenges/uncertainties
N/A

### Checks PR readiness
- [ ] UI: works on smaller screen sizes
- [ ] UI: feedback for any loading/error states
- [ ] Check cancel/go-back flows
- [ ] Check database state correct when deleting/updating (especially regarding relationships)
- [x] changelog
- [x] npm lint
- [x] no new deprecations

### CODE REVIEW

File: test-app/src/multiple-growing-editors.js [hunk 1]
```diff
@@ -7,7 +7,10 @@ const decisionType =
   'https://data.vlaanderen.be/id/concept/BesluitType/0d1278af-b69e-4152-a418-ec5cfd1c7d0b';
 const options = {
   besluit: { decisionUri },
-  lpdc: { endpoint: 'http://localhost/lpdc-service', decisionUri },
+  lpdc: {
+    endpoint: 'http://embeddable.dev.gelinkt-notuleren.lblod.info:7070',
```
Comment (abeforgit) [inline test-app/src/multiple-growing-editors.js:11]: not clear why this one is different

**Review by abeforgit — CHANGES_REQUESTED** [review]
was a bit puzzled how to test this one, you mention "should work locally" but it can't, since the frontend references an absolute url, and the compose config assumes a running nginx-proxy stack
I did try to deploy this branch on the dev server (saving the old config under BACKUP-app-gn-embeddable-dev)
and with that running the frontend with `pnpm start` does work, so the url is reachable

I'm tempted to merge to get it out of our way since its just a dev setup, but we should revisit this.

File: test-app/src/multiple-growing-editors.js (hunk 1 above)
Comment (piemonkey) [reply test-app/src/multiple-growing-editors.js:11]: Oh, that was for a previous version where I just published a port. I'll change it.

**Comment by piemonkey** [comment]
Ah, by locally 'should just work', I meant with the config changed to point to local. Not exactly 'just working' now that I look at it again... I should also have added that there are 2 parts to testing this, one which can be tested locally, which is just showing that the proxy works without CORS issues and the other that the docker-compose config works.

It wouldn't be hard to add in a proxy using the image used on the server to test the whole set-up locally, but thought that was overkill.

**Pull request merged**